chapters/connectors.tex

```diff
@@ -813,7 +813,9 @@ \subsection{Overconstrained Equation Operators for Connection Graphs}\label{over
   end equalityConstraint;
 end Record;
 \end{lstlisting}
-The \lstinline!residue! output of the \lstinline!equalityConstraint! function shall have known size, say constant $n$.  The function shall express the equality between the two type instances \lstinline!T1! and \lstinline!T2! or the record instances \lstinline!R1! and \lstinline!R2!, respectively, with a non-redundant number $n \ge 0$ of equations.  The residues of these equations are returned in vector \lstinline!residue! of size $n$.  The set of $n$ non-redundant equations stating that \lstinline!R1 = R2! is given by the equation (\lstinline!0! represents a vector of zeros of appropriate size):
+The array dimension $n$ of \lstinline!residue! shall be a constant \lstinline!Integer! expression that can be evaluated during translation, with $n > 0$.
```

Comment:
Sorry, I am too late with my review.  I see the restriction of 'n > 0' was added. There is at least one Modelica library [that sets it to 0](https://github.com/lbl-srg/modelica-buildings/blob/bddfd49d63a9c056073551b1267591be680e4852/Buildings/Electrical/PhaseSystems/PartialPhaseSystem.mo#L30).  @mwetter

Reply:
Excellent observation.  Will fix.

Reply:
Having opened #2909, I'll copy the comments above to that conversation and mark this conversation as resolved.